### ISSUE TITLE
ADA compliant issue

### ISSUE THREAD

**Issue by SayedTareq** [CalderaWP/Caldera-Forms#3686]
My ADA team found these issues on Caldera Form

Notification not announced - Loading notification is not announced by the screen reader.
Status message not announced - The screen reader doesn't announce the status message when user triggers the 'Contact' button with valid details.
